src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -6,32 +6,35 @@ import { SearchApiPath } from '../../../../api/apiPaths';
 import { SearchResponse } from '../../../../types/common.types';
 import { ListSkeleton } from '../../../../components/ListSkeleton';
 import { CentralImportResultItem } from './CentralImportResultItem';
-import { ResourceFieldNames } from '../../../../types/publicationFieldNames';
+import { DescriptionFieldNames, ResourceFieldNames } from '../../../../types/publicationFieldNames';
 import { ROWS_PER_PAGE_OPTIONS } from '../../../../utils/constants';
-import { createSearchQuery, ExpressionStatement, SearchConfig } from '../../../../utils/searchHelpers';
+import { DuplicateSearchFilters } from './DuplicateSearchFilterForm';
 
 interface CentralImportDuplicateSearchProps {
-  publication: Registration;
+  duplicateSearchFilters: DuplicateSearchFilters;
 }
-export const CentralImportDuplicateSearch = ({ publication }: CentralImportDuplicateSearchProps) => {
+
+export const CentralImportDuplicateSearch = ({ duplicateSearchFilters }: CentralImportDuplicateSearchProps) => {
   const { t } = useTranslation('basicData');
 
-  const searchConfig: SearchConfig = publication.entityDescription?.reference?.doi
-    ? {
-        properties: [
-          {
-            fieldName: ResourceFieldNames.Doi,
-            value: publication.entityDescription.reference.doi,
-            operator: ExpressionStatement.Contains,
-          },
-        ],
-      }
-    : {};
+  const maxHits = ROWS_PER_PAGE_OPTIONS[0];
+  const queryArray = [];
+  duplicateSearchFilters.doi.length > 0 && queryArray.push(`${ResourceFieldNames.Doi}:"${duplicateSearchFilters.doi}"`);
+  duplicateSearchFilters.title.length > 0 &&
+    queryArray.push(`${DescriptionFieldNames.Title}:"${duplicateSearchFilters.title}"`);
+  duplicateSearchFilters.author.length > 0 &&
+    queryArray.push(`entityDescription.contributors.identity.name:"${duplicateSearchFilters.author}"`);
+  duplicateSearchFilters.issn.length > 0 &&
+    queryArray.push(`reference.publicationContext.printIssn:"${duplicateSearchFilters.issn}"`);
```

Comment:
mangler det ikke en `entityDescription` her?
```suggestion
    queryArray.push(`entityDescription.reference.publicationContext.printIssn:"${duplicateSearchFilters.issn}"`);
```